src/main/java/org/wildfly/security/http/impl/ClientCertAuthenticationMechanism.java

```diff
@@ -83,39 +83,39 @@ public void evaluateRequest(HttpServerRequest request) throws HttpAuthentication
         Function<SecurityDomain, IdentityCache> cacheFunction = createIdentityCacheFunction(request);
 
         if (cacheFunction!= null && attemptReAuthentication(request, cacheFunction)) {
-            log.trace("ClientCertAuthenticationMechanism: re-authentication succeed");
+            httpClientCert.trace("Re-authentication succeed");
             return;
         }
         if (attemptAuthentication(request, cacheFunction)) {
             return;
         }
-        log.trace("ClientCertAuthenticationMechanism: both, re-authentication and authentication, failed");
+        httpClientCert.trace("Both, re-authentication and authentication, failed");
         fail(request);
     }
 
     private boolean attemptAuthentication(HttpServerRequest request, Function<SecurityDomain, IdentityCache> cacheFunction) throws HttpAuthenticationException {
         Certificate[] peerCertificates = request.getPeerCertificates();
         if (peerCertificates == null) {
-            log.trace("CLIENT-CERT Peer Unverified");
+            httpClientCert.trace("Peer Unverified");
             request.noAuthenticationInProgress();
             return true;
         }
 
         X509Certificate[] x509Certificates = X500.asX509CertificateArray(peerCertificates);
         X509PeerCertificateChainEvidence evidence = new X509PeerCertificateChainEvidence(x509Certificates);
 
-        log.tracef("Using ClientCertAuthenticationMechanism to authenticate the following certificates: [%s]", x509Certificates);
+        httpClientCert.tracef("Using ClientCertAuthenticationMechanism to authenticate the following certificates: [%s]", x509Certificates);
```

Comment:
again: no need to mention ClientCertAuthenticationMechanism when specified by logger: "Authenticating using following certificates:" will be sufficient

Reply:
Sorry, I somehow overlooked it